cumulus/parachains/runtimes/assets/asset-hub-rococo/Cargo.toml

```diff
@@ -36,7 +36,7 @@ pallet-nfts = { path = "../../../../../substrate/frame/nfts", default-features =
 pallet-nfts-runtime-api = { path = "../../../../../substrate/frame/nfts/runtime-api", default-features = false }
 pallet-proxy = { path = "../../../../../substrate/frame/proxy", default-features = false }
 pallet-session = { path = "../../../../../substrate/frame/session", default-features = false }
-pallet-state-trie-migration = { path = "../../../../../substrate/frame/state-trie-migration", default-features = false, optional = true }
+pallet-state-trie-migration = { path = "../../../../../substrate/frame/state-trie-migration", default-features = false }
```

Comment:
This should still be gated by the construct_runtime! feature gate. But i think that is not actually working, otherwise this change would not be needed 🤷‍♂️

Reply:
It is still deployed though, will remove it then.

Reply:
Uhf, did we really genesis the asset-hub-rococo with state_version 0?  
I just queried `Core::version` of the 1.1 runtime and it returns state_version=0...

Looks like it was copy&pasted from AHK. @cheme can you double-check please?

Reply:
Ouch https://github.com/polkadot-fellows/runtimes/blob/31ba26287ec752574244f0d690167f7ae8430c8b/system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs#L153 . (kusuma is fine).
Statemint on rococo (I guess it is now assethub, not sure) was migrated IIRC or was it only on westend cc @PierreBesson ?

Reply:
I meant it here for the Asset-hub-rococo: https://github.com/paritytech/polkadot-sdk/blob/5601f2865b3f867131654189c08e7776baf83d16/cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs#L110

Not sure, since when i queried the runtime api it returned version 0. Maybe we can check with the RPC but the public node does not support it.

Reply:
maybe https://github.com/polkadot-fellows/runtimes/issues/74